Assets/Scripts/SceneControllers/LevelSelectController.cs

```diff
@@ -57,12 +59,14 @@ protected void Start()
 
         // Initialize levels (unlocked etc.)
         LoadLevels();
+
+        isMovingIcon = false;
     }
 
     override protected void SceneUpdate()
     {
         // Enter level
-        if (Input.GetKeyDown(KeyCode.Return))
+        if (Input.GetKeyDown(KeyCode.Return) && !isMovingIcon)
```

Comment:
Can we add this check to the left and right arrow actions too? I think it'll feel better

Reply:
oooo good call